presto-kafka/src/main/java/io/prestosql/plugin/kafka/encoder/raw/RawRowEncoder.java

```diff
@@ -168,7 +168,7 @@ private static int parseOffset(String group, String offsetName, String columnNam
         private static FieldType parseFieldType(String dataFormat, String columnName)
         {
             try {
-                if (!dataFormat.equals("")) {
+                if (dataFormat != null && !dataFormat.equals("")) {
```

Comment:
I think it would be nice to revisit the KafkaColumnHandle/KafkaTopicFieldDescription and make use of Optionals where it makes sense to give a hint of what our spec requires. Right now you cannot depend on the code without consulting the documentation.

Reply:
Please add an issue for this.

Reply:
https://github.com/prestosql/presto/issues/5898